### ISSUE TITLE
Add running club meeting point POI

### ISSUE THREAD

**Issue by Eelviny** [organicmaps/organicmaps#7055]
**Is your feature request related to a problem? Please describe.**
I frequently take part in weekly organised running events. They have a set location where they start each week, and I can find them easily on Google Maps. However as they're usually hosted in an outdoor public location like a park they have no building or amenity, so they don't show on Organic Maps.

**Describe the ideal solution**
I'd love to have POI for running club meeting points. See here: https://overpass-turbo.eu/?template=key-value&key=club&value=sport. Most of them have opening hours indicating the time at which it's "open" i.e. the run is taking place.

Basically add `sport=running`.

**Comment by pastk** [comment]
Yeap we don't support yet `club=sport` and `sport=running`.
Also need a suitable icon.

@Eelviny any exact coordinates with functioning running clubs, please?

**Comment by Eelviny** [comment]
Sure, here's a couple different examples:

- https://www.openstreetmap.org/node/11473096778 `52.3272105, 4.8550672` - example of an event hosted in a public area
- https://www.openstreetmap.org/way/621213966 `53.7768076, -0.2818398` - example of a running clubhouse dedicated building
- https://www.openstreetmap.org/node/6010143371 `51.498524, -0.0730964` - this is a tricky one as it's marked as `amenity=bar` but it's just this is their clubhouse where they tend to have drinks. I assume they did this to make it appear on a map.

Performing a query with this combination of tags `club=sport|sport=running|amenity=*` gave me 3 results - 1 bar, 2 community centres. I would suggest overriding the amenity if the other tags are set.
